draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -261,6 +261,11 @@ If implemented it MUST use the following rules:
 
 [^may-should-trustbase]: Open discussion: RFC6614 says "may" here. I think this should be a "should". There are some discussions to change this to "must". Input from TLS/UTA experts is appreciated.
 
+If this model is implemented, RADIUS/(D)TLS clients SHOULD NOT be pre-configured with a list of known public CAs by the vendor or manufacturer. Instead, the clients SHOULD start off with an empty CA list. The addition of a CA SHOULD be done only when manually configured by an administrator.
```

Comment:
I would suggest amending this to reflect should not be pre-configured as *already* trusted for the client.

I could perceive an onboarding process that assists the administrator to know which CA to add as a trust anchor for the given client.